cfme/tests/services/test_provision_stack.py

```diff
@@ -341,15 +329,15 @@ def test_retire_stack(provider, provisioning, catalog, catalog_item, request):
     DefaultView.set_default_view("Stacks", "Grid View")
 
     stack_data = prepare_stack_data(provider, provisioning)
-    service_catalogs = ServiceCatalogs("service_name", stack_data)
-    service_catalogs.order_stack_item(catalog.name, catalog_item)
+    service_catalogs = ServiceCatalogs(item_name, stack_data)
+    service_catalogs.order()
     logger.info('Waiting for cfme provision request for service %s', item_name)
     row_description = item_name
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
                        fail_func=requests.reload, num_sec=2500, delay=20)
 
-    assert provision_success_message(catalog_item.name) in row.last_message.text
+    assert 'Provisioned Successfully' in row.last_message.text
```

Comment:
![screenshot from 2017-01-09 18 19 54](https://cloud.githubusercontent.com/assets/3866976/21795963/4d88a2cc-d6ba-11e6-8715-06c0cbea1caa.png)


Last message is not same for all requests.hence checking 
"Provisioned successfully " only